tests/hooks_test.py

```diff
@@ -121,3 +121,127 @@ def test_python_no_log_warn_positive(s):
+        r"` /dFm?k9lC'U;-DZ;ltgWSUJ3CH(V. 6 \p?vZ-Ln@IQ.>&nIFx,aP+(pNjSDf:s3CblBJzob>%Ua.Js`0",
+        r"`PbW8bYacjkh;|L/y^W fca{Da4:G'et?%bX(1z?`",
+        r"`]rHa{^-*1g!aH$tJ'!5~V6HMU!)c_\"wCuPBQtLY+u>tktX%.u7F5;14>J`",
+        r"`\"UgYN@5K0'x]\"vRVDi*cnUOaIu f=o0#ii-MmXOk;d4:n5z6X\"}uba9'?Epf\"gfJIf7yc <V5<yzq%rl%R\3`5",
+        r"`Q6wBD}MJ*(\"d(~8+Ka<%+%lj>[9O}`-",
+        r" `J/GmI^pI}$v>,WpRfu5pYS-:|^tONqjsA[.e.4l^{%/ubuGpVzVkuR#UgJdC3`!",
+        r"`&B%aYM1(IDu;'>+Nbloz?5Ls-7=9z:vrtIi7y.z(J2-D]o0^o`b",
+        r"`(^;[ U3hvu}}&?Y=^=6rxt|y|6WX,{#)pwO{X9!`",
+        r"`nEC8OnuMjv/J$&)r^`",
+        r" `Tm\_BZoJt\eR^*<WL'0_kI>=L#i%V7L$FmzYDQ_>4\"W)FAZ69i-^*KLxnsX.K1JWYW4v*iH,^wV|gO2rd]bNmlxADPTf K0g`7",
+        r"`W0]AZp@I9V^)kv]Q+>E,i6~Sz>49Wa)#9LRg9(3aR*arU\"[Y[&m,zql}<55n_hU7\c0VP?1!IGdU2O+igaX\w/'JXitmA]6N`",
+        r" `?S/D7N<c!*?&$=IY@-{2n\"DphNh;g'\d\"^ )zO/r|cN%6`5",
+        r" `f\jG\|7&Gr/mC#CZERlxUu!^;`",
+        r" `)<}BT<?oS~F>\")&asY9(-Wsckh%rr}j4?[q&7gh]XCFCKq=@n7y2khE.1/@,1coAzNWgNmMz7[-{\"i`",
+        r"`Uu8e<kx/M&kB)0!,XMsb[[kX;2P.:j|+7`z",
+        r" `,bQar6DP$hT^'w|^00d1}Z@IWYb6D%?uA?>DR,K&iz@^eh>}Li*:Niz1aOrvGM?h>[?F5r/ti@QxC%IoK|* %\1\y'5gS`",
+        r" `^.MFOhA;o1RG,a:5+m?,`",
+        r"`.yCTvV<s&*=smX-9*%:Xz`s",
+        r" `RJ`",
+        r" `]s&&5b^@0|\"E]Pz^z]# @:X];FIXrngifU1]u'K{J/eSpV&F}Jz#}( l*4P:ur^Q}Q2N0eKF$W:;SKld#7sr~lR`",
+        r"`)ny6*/yW2$>7YY~{:#Ajrox,p[jwF_wC3<XQLA,)$j<6' KF7!B5.sK#B!Uj,h2UE$kW]HO:L}R/mO`",
+        r" `_Zj5\=%MF<&o_\"wgq002f})\,U^O'w8Dx8{IOi].t(2O4WYP:1uKgtgC0 4cky>]~Sf%^`",
+        r"`bs?u+)RCNOpgC(UM3&XSMtx8)_j+i>$+e,#sj|x%`",
+        r"` 7K*:9t^B]aU,_qVGe5x5h&JO4!m:GW3FDym=qhVr<ZRRM4Pq'VG7\GNT2Jn2.mk#c/PCzBr4m*)\0 Au]TxcW7j\"|`t",
+        r" `eOm$mFN1-dMjx\"36G=6{Z/<Tu+CkO`d",
+        r" `hbNYJH)=[qM &WCoqm6WNEt1b.O.mwL%TSIppo^<<_gQ3Z;kZ8\0w`D",
+        r"`mPgA/(tvL#4;GA.SlVBR3+>tml@@xb*3]/K>2P}iG;s|HR.?/hi#OmaADLW9Wq7K dtIg[gY1K#p)[s8nj~|xU2`u",
+        r" `<J+HyZ $X-k6DXc-n\S,8qOw2xDVJ%.Y~iK:MO 3}<Hx2n+aJKlS;A[:;zkaJb0X,Dyc~Y^oOH%R#`",
+        r" `lm+P3z&Q7\XC*!9X\@&Pl4ubZ\"~Qk3yINB#~OEvm[ck6P18(9)y!'2,'m)~j/gTP-e)HV7<dl>v//>Cz[U^ bGo:0_^`",
+        r"`L_U!Hc1uz|-MstBD$/W&Uq)8qg1q)A(q`",
+        r" `*|]%h6O!(g,hNA|>w_kce>d7exP#K4f'%}21;A3q64&UVP,5xAo\"!KrW;Gx3qt77rn|\+ri@S?mNE/du<?4.f|c/%cX+}*A`Z",
+        r"`~rLE!9Yxn_THQr>?P/=@6\#yY`P",
+        r" `>yf2x-aRXe;GQ{x-!3Yy=6P%9o$7BM=a9v<y7!'N.h?aK;]4 6Ia\Ox, ;z}7$0@-=zd*r,_#CCNd<cBujRVB6oVhCu:cQV}*6=i`Y",
+        r"`T94c#wS6S>C-LQO71Qkas#4_l%%5I=:lGC)[R0c^qGc& 6{2X}?y]~IBpbyZY;`Z",
+        r" `)6*I?95H!$*l0p~\"3C*&]8O?@;R7Tvb>?4|AaV:A??:Um2{Lv|d5X=jFxr;8 *`-",
+        r"`GACiD1xA1r9u*&}HP_f];r5~-]}EXCUhPrpDzy`",
+        r" `JPdzQ2QqF_AQl=t]`",
+        r" `97P}j:' :MJhH,JnFCVlE\"pX cM/j6!s:=69Wf>Yc[5mb9z)W?Hb5m{Lt,)f,&x<mr'C3NBDMnr6<'|-b6`>",
+        r" `@ ~7JyWy/V9V<th:}y.#``S",
+        r" `- 3]1f{I**4Q).6}/#BrEPW.2^v]4n6oia8Xu_fOgh:q_E9Qxp}n:p T)*EId99,[7w^&AX(uQ_7~$)DX*z=t].qLh+ON>sMOt}`",
```

Comment:
I'm not interested in this test, please remove it -- tests should be obvious from reading what they are testing -- this one is nonsense

Reply:
Removed